Physical_Engine/Query/SchedulingRadius.cs

```diff
@@ -0,0 +1,59 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.ComponentModel;
+using BH.oM.Reflection.Attributes;
+using BH.oM.Quantities.Attributes;
+
+namespace BH.Engine.Physical
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        [Description("Gets the minimum scheduling radius based on the diameter of the reinforcement bar using the values given in BS 8666:2020 Table 2.")]
+        [Input("diameter", "The diameter of the reinforcement bar to determine the minimum scheduling radius.")]
+        [Output("radius", "The minimum scheduling radius based on the diameter of the reinforcement bar", typeof(Length))]
+        public static double SchedulingRadius(this double diameter)
```

Comment:
I think this method should take an input representing the code it complies to. This means we would need an enum containing the codes that we support, plus the whole system would need to be tweaked to pass this value around seamlessly. Not sure how to marry this with my comment on https://github.com/BHoM/BHoM/pull/1286 regarding the shape code namespaces.

What do you think @peterjamesnugent?

Reply:
As discussed the input will be a `Reinforcement` and the `ShapeCode` used to determine the standard used.